### PR TITLE
docs(common): add kb for read-only properties

### PR DESCRIPTION
**Pull request by YanushevGeorgi**
Related to: [#7373](https://github.com/telerik/blazor/issues/7373)

### CODE REVIEW

File: knowledge-base/common-readonly-properties [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: Error when using read-only model properties and editing
```
Comment (yordan-mitev) [inline knowledge-base/common-readonly-properties:2]: Use title case. See the [capitalization rules](https://docs.telerik.com/style-guide/titles-and-headings#capitalization).

File: knowledge-base/common-readonly-properties [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            <td>Product</td>
+            <td>Grid for Blazor, <br /> Scheduler for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+Editing does not work, and the OnUpdate event does not fire when a property in my bound model does not have a setter or is read-only.
```
Comment (yordan-mitev) [inline knowledge-base/common-readonly-properties:26]:
```suggestion
Editing does not work, and the `OnUpdate` event does not fire when a property in my bound model does not have a setter or is read-only.  
```

File: knowledge-base/common-readonly-properties [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+## Description
+
+Editing does not work, and the OnUpdate event does not fire when a property in my bound model does not have a setter or is read-only.  
+
+## Error Message
+
+If the Telerik source code is attached to the app, the following exception is thrown: 
+ - `System.ArgumentException`: Property set method not found.
+
+## Cause/Possible Causes
```
Comment (yordan-mitev) [inline knowledge-base/common-readonly-properties:33]: This is a templated heading, choose the most appropriate option for this KB.
```suggestion
## Cause
```

File: knowledge-base/common-readonly-properties [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+title: Error When Using Read-Only Model Properties and Editing
+description: The editing fails when the bound model has read-only properties
+type: troubleshooting
+page_title: Error When Using Read-Only Model Properties and Editing
+slug: common-kb-read-only-properties
+position: 
+tags: grid, scheduler, edit, telerik, blazor
+ticketid: 1631052
```
Comment (dimodi) [inline knowledge-base/common-readonly-properties:9]: Don't we have other tickets for this too?

File: knowledge-base/common-readonly-properties [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+</table>
+
+
+## Description
+
+Editing does not work, and the `OnUpdate` event does not fire when a property in my bound model does not have a setter or is read-only.  
+
+## Error Message
+
+If the Telerik source code is attached to the app, the following exception is thrown:
```
Comment (dimodi) [inline knowledge-base/common-readonly-properties:30]:
```suggestion
If the Telerik source code is attached to the app, the following exception is thrown: 

```

**Review by dimodi — APPROVED** [review]

**Pull request merged**